packages/native-stack/src/views/NativeStackView.native.tsx

```diff
@@ -201,6 +207,19 @@ const SceneView = ({
     freezeOnBlur,
   } = options;
 
+  const Screen = React.useContext(ScreenContext);
```

Comment:
This line has been added here, because functional components cannot have ref assigned. That's why we're doing a pretty tricky thing here and we're using a `ScreenContext,` which uses `InnerScreen` by default, where `InnerScreen` uses forwardRef under the hood.